### ISSUE TITLE
Ember installation failed

### ISSUE THREAD

**Issue by romaef** [noppo-pro/ember-quill#3]
When i do `ember install ember-quill` it gives me the error spawn undefined ENOENT

**Comment by pcambra** [comment]
I'm getting "npm ERR! cb() never called!"

**Comment by epicmiller** [comment]
I'm having the same `npm ERR! cb() never called!` problem. Doesn't seem to matter how clean of an environment I start with, or what version of node/npm I use (tried node 5, 6 and 7).

**Comment by cspanring** [comment]
Experiencing the same issue with yarn, which is a little bit more explicit:

```
error An unexpected error occurred: "https://registry.yarnpkg.com/ember-quill/-/ember-quill-0.0.6.tgz: Invalid tar header. Maybe the tar is corrupted or it needs to be gunzipped?".
```
maybe something went wrong during publishing? maybe a re-publish could solve the issue?

**Comment by cspanring** [comment]
must be a publishing blip, I forked the repo, published it under my scope and successfully installed it as a workaround:
```
yarn add @cspanring/ember-quill --dev
```

**Comment by jgelens** [comment]
Fixed with #2
